src/from-attribute.js

```diff
@@ -105,10 +105,28 @@ canReflect.assignSymbols(ElementAttributeObservable.prototype, {
 
 module.exports = function fromAttribute (propertyName) {
 	return function fromAttributeBind (instance) {
-		return new Bind({
+		const bind = new Bind({
 			parent: new ElementAttributeObservable(instance, propertyName, {}, 'attributes'),
 			child: value.to(instance, propertyName),
 			queue: "domUI"
 		});
+
+		// This is to prevent canReflect from reading the parent value and setting the child
+		// when the parent / instance doesn't isn't an attribute
```

Comment:
> doesn't isn't

Reply:
Sorry wasn't able to get clear english down